release.wdl

```diff
@@ -0,0 +1,238 @@
+#Thoughts--
+#Still need to manually check that the jars are being created in broad artifactory
+#Need to manually investigate if tests aren't being written
+#Need to manualy update the release portion of Cromwell
+#Need to make sure that the changelog.md file doesn't get weirdly altered when merging to master for Cromwell
+
+
+task basicRelease {
+   Array[String] dependency
+   String repo
+   String releaseV
+   String baseDir
+
+   command {
+     cd ${baseDir}/${repo}
+     git checkout develop
+     git pull
+
+     currentV=`grep "git.baseVersion " build.sbt | cut -d "=" -f 2 | tr -d \"\,`
+     echo "Version of ${repo} on the develop branch is $currentV"
+
+     if [ ${releaseV} -ne $version ]; then
+        echo "Need to update the ${repo} version on the develop branch to ${releaseV}" >&2
+        sed -i -e 's/'"$currentV"'/'"${releaseV}"'/' build.sbt
```

Comment:
Where environment is this script running? `sed` on macOS treats `-i` differently than say linux, and I didn't see a docker image specified.

http://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux

Same goes for other `sed`s below.